extensions/arc/src/localizedConstants.ts

```diff
@@ -29,7 +29,7 @@ export const newSupportRequest = localize('arc.newSupportRequest', "New support
 export const diagnoseAndSolveProblems = localize('arc.diagnoseAndSolveProblems', "Diagnose and solve problems");
 export const supportAndTroubleshooting = localize('arc.supportAndTroubleshooting', "Support + troubleshooting");
 
-export const createNew = localize('arc.createNew', "Create New");
+export const newInstance = localize('arc.createNew', "New Instance");
```

Comment:
Guess some extra cleanup changes snuck in here - whoops. Oh well, it's pretty much just renaming the button to be more descriptive of what it's doing.